README.md

```diff
@@ -16,30 +16,42 @@ Delightful code generation for OpenAPI specs for Swift written in Swift.
 ### [Mint](https://github.com/yonaskolb/Mint)
 
 ```bash
-mint install CreateAPI/CreateAPI
+$ mint install CreateAPI/CreateAPI
 ```
 
-Usage:
+### [Homebrew](https://formulae.brew.sh/formula/create-api)
 
 ```bash
-mint run CreateAPI create-api generate -h
+$ brew install create-api
 ```
 
+### Swift Package Plugins
+
+- [Creating a Swift Package Plugin](./Docs/SwiftPackagePlugins.md)
+
```

Comment:
Now that we're available through Homebrew, should have installation instructions.

Reply:
Clicked the wrong line, woops.

Reply:
It's just above this line already 😄

Reply:
Oh, apologies I missed it.